test.js

```diff
@@ -48,6 +48,50 @@ test('wrap core method', async t => {
 	t.is(JSON.parse(await fn(fs.readFile)('package.json')).name, 'pify');
 });
 
+test('binds to the original object by default', async t => {
+	const obj = {
+		x: 'foo',
+		y: function (cb) {
+			setImmediate(() => cb(null, this.x));
```

Comment:
I don't know if it's preferred, but `setImmediate` will pass on any extra arguments, so this could be written like this:

```js
setImmediate(cb, null, this.x);
```